.github/dependabot.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+updates:
+  # Enable version updates for Gradle
+  - package-ecosystem: "gradle"
+    directory: "/"
+    groups:
+      # We group minor and patch updates together because they are less likely to break things.
+      # Major updates will be PR-ed individually: they are more likely to need developer
+      # intervention.
+      minor-patch-dependencies:
+        patterns:
+          - "*"
+        exclude-patterns:
+          - "com.diffplug.spotless" # likely to require reformatting of code
+        update-types:
+          - "minor"
+          - "patch"
+      # We want Swagger codegen dep versions to stay in sync with each other, and be PRed together.
+      io.swagger.codegen.v3:
+        patterns:
+          - "io.swagger.codegen.v3"
+    schedule:
+      interval: "weekly"
```

Comment:
Do we want to go ahead and move this to a less frequent cadence as discussed in retro?